core/src/main/java/org/jdbi/v3/core/internal/JdbiClassUtils.java

```diff
@@ -221,15 +222,15 @@ public static <T> MethodHandleHolder<T> findConstructor(Class<T> type, Class<?>.
     }
 
     private static MethodHandle findCtorMethodHandleForParameters(Class<?> type, Class<?>... types) {
-        var suppressedThrowables = new LinkedList<Throwable>();
+        Deque<Throwable> suppressedThrowables = new ArrayDeque<>();
```

Comment:
some reason `ArrayList` is not good enough here?

Reply:
yes, it uses the deque/linked list methods.